edb/graphql/types.py

```diff
@@ -437,14 +437,8 @@ def _convert_edb_type(
     ) -> Optional[GraphQLOutputType]:
         target: Optional[GraphQLOutputType] = None
 
-        # only arrays can be validly wrapped, other containers don't
-        # produce a valid graphql type
         if isinstance(edb_target, s_types.Array):
             subtype = edb_target.get_subtypes(self.edb_schema)[0]
-            if str(subtype.get_name(self.edb_schema)) == 'std::json':
```

Comment:
Use `issubtype` here

Reply:
These are deleted lines, so in the new version we don't perform that check. Subtype is used for the recursion.